src/components/LoginForm.jsx

```diff
@@ -0,0 +1,223 @@
+import React from "react";
+import Form from "./FormBuilder";
+
+const FORM_CONFIG = {
+  onSubmit: () => {
+    console.log("submit");
+    alert("hey");
+  },
+  wrapperClass: "dd",
+  id: "contact-form",
+  title: { text: "Form", class: "ll" },
+  submitBtn: {
+    id: "submit-form-btn-id",
+    content: "Submit",
+    class: "sub"
+  },
+  fields: {
```

Comment:
Can we keep fields as array, that would save us a lot of rewrite ?

Reply:
no need